core/src/main/scala/com/raphtory/service/ClusterManagerService.scala

```diff
@@ -4,19 +4,14 @@ import cats.effect.ExitCode
 import cats.effect.IO
 import cats.effect.IOApp
 import cats.effect.Resource
-import com.raphtory.Raphtory
-import com.raphtory.Raphtory.makeLocalIdManager
-import com.raphtory.internals.communication.connectors.AkkaConnector
-import com.raphtory.internals.communication.repositories.DistributedTopicRepository
 import com.raphtory.internals.components.RaphtoryServiceBuilder
-import com.raphtory.internals.management.ZookeeperConnector
-import com.raphtory.internals.management.arrow.ZKHostAddressProvider
+import com.raphtory.internals.management.GraphConfig.ConfigBuilder
 
 object ClusterManagerService extends IOApp {
 
   def run(args: List[String]): IO[ExitCode] =
     (for {
-      config  <- Resource.pure(Raphtory.getDefaultConfig())
+      config  <- Resource.pure(ConfigBuilder().build().getConfig)
       service <- RaphtoryServiceBuilder.manager[IO](config)
```

Comment:
you don't need this, just make a `val` at the start of the `run` function and pass it down to `manager[IO](config)`